templates/init/firestore/firestore.rules

```diff
@@ -1,16 +1,14 @@
 service cloud.firestore {
   match /databases/{database}/documents {
-    match /{document=**} {
```

Comment:
we want to match inside a collection, there are no top level documents in firestore

can you also make the change to RTDB and storage rules?

Reply:
Done